package.json

```diff
@@ -58,7 +58,6 @@
     "jest-image-snapshot": "^2.7.0",
     "jest-puppeteer": "^3.5.1",
     "pify": "^4.0.1",
-    "portfinder": "^1.0.19",
```

Comment:
please leave this in i'm using it on the pre-rendering branch

Reply:
oh, why not use the value in `.env.test`?

Reply:
yea pre-rendering doesn't need to be interacted with i'm just starting the server to use the proxy config so i figured using a made up port would be preferable there.